frontend/packages/dev-console/src/components/topology2/components/edges/BaseEdge.tsx

```diff
@@ -32,7 +49,7 @@ const BaseEdge: React.FC<BaseEdgeProps> = ({
   return (
     <Layer id={dragging || hover ? 'top' : undefined}>
       <g
-        ref={hoverRef}
+        {...editAccess && { ref: hoverRef }}
```

Comment:
What does `hover` have to do with `editAccess`?!
We need `hover` to provide a hover style treatment.
You need to prevent showing the connector if the user doesn't have edit access.
Furthermore, editing the visual connector edits the `source` node while the service request binding connector creates a new SBR. Two completely different access control scenarios that require different checks.

Reply:
I do check ```editAccess``` here because the delete connection option appears on hover.

Reply:
But we do other things on hover such as changing the color of the line to be blue and raised.

Reply:
removed the ```editAccess``` checked from ```hoverRef```.